agents-api/agents_api/routers/docs/embed.py

```diff
@@ -22,6 +22,6 @@ async def embed(
         [text_to_embed] if isinstance(text_to_embed, str) else text_to_embed
     )
 
-    vectors = await litellm.aembedding(inputs=text_to_embed)
+    vectors = await litellm.aembedding(inputs=[data.embed_instruction] + text_to_embed)
```

Comment:
Why are you embedding the instructions separately? This looks like a but

Reply:
You're right. Pushed a fix